src/main/resources/assets/croptopia/lang/en_nz.json

```diff
@@ -0,0 +1,632 @@
+  "item.croptopia.pecan_ice_cream": "Pecan Ice Cream",
+  "item.croptopia.pecan_pie": "Pecan Pie",
+  "item.croptopia.protein_bar": "Protein Bar",
+  "item.croptopia.raisin_oatmeal_cookie": "Raisin Oatmeal Cookie",
+  "item.croptopia.roasted_nuts": "Roasted Nuts",
+  "item.croptopia.trail_mix": "Trail Mix",
+
+  "item.croptopia.burrito": "Burrito",
+  "item.croptopia.tostada": "Tostada",
+  "item.croptopia.horchata": "Horchata",
+  "item.croptopia.carnitas": "Carnitas",
+  "item.croptopia.fajitas": "Fajitas",
+  "item.croptopia.enchilada": "Enchilada",
+  "item.croptopia.churros": "Churros",
+  "item.croptopia.tamales": "Tamales",
+  "item.croptopia.tres_leche_cake": "Tres Leche Cake",
+  "item.croptopia.stuffed_poblanos": "Stuffed Poblanos",
+  "item.croptopia.chili_relleno": "Chili Relleno",
+  "item.croptopia.crema": "Crema",
+  "item.croptopia.refried_beans": "Refried Beans",
+  "item.croptopia.chimichanga": "Chimichanga",
+  "item.croptopia.quesadilla": "Quesadilla",
+
+  "item.croptopia.cinnamon": "Cinnamon",
+  "item.croptopia.corn_husk": "Corn Husk",
+  "item.croptopia.whipping_cream": "Whipping Cream",
+  "item.croptopia.vanilla_seeds": "Vanilla Seeds",
+
+  "item.croptopia.cinnamon_sapling": "Cinnamon Sapling",
+  "item.croptopia.cinnamon_log": "Cinnamon Log",
+  "item.croptopia.stripped_cinnamon_log": "Stripped Cinnamon Log",
+  "item.croptopia.cinnamon_wood": "Cinnamon Wood",
+  "item.croptopia.stripped_cinnamon_wood": "Stripped Cinnamon Wood",
+
+  "item.croptopia.shepherds_pie": "Shepherd's Pie",
+  "item.croptopia.beef_wellington": "Beef Wellington",
+  "item.croptopia.fish_and_chips": "Fish and Chips",
+  "item.croptopia.eton_mess": "Eton Mess",
+  "item.croptopia.tea": "Tea",
+  "item.croptopia.cornish_pasty": "Cornish Pasty",
+  "item.croptopia.scones": "Scones",
+  "item.croptopia.figgy_pudding": "Figgy Pudding",
+  "item.croptopia.treacle_tart": "Treacle Tart",
+  "item.croptopia.sticky_toffee_pudding": "Sticky Toffee Pudding",
+  "item.croptopia.trifle": "Trifle",
+  "item.croptopia.water_bottle": "Water Bottle",
+  "item.croptopia.milk_bottle": "Milk Bottle",
+  "item.croptopia.tea_leaves": "Tea Leaves",
+  "item.croptopia.tea_seed": "Tea Seeds",
+
+  "item.croptopia.ajvar": "Ajvar",
+  "item.croptopia.ajvar_toast": "Ajvar Toast",
+  "item.croptopia.avocado_toast": "Avocado Toast",
+  "item.croptopia.baked_sweet_potato": "Baked Kūmara",
+  "item.croptopia.baked_yam": "Baked Yam",
+  "item.croptopia.beef_stew": "Beef Stew",
+  "item.croptopia.beef_stir_fry": "Beef Stir Fry",
+  "item.croptopia.buttered_green_beans": "Buttered Green Beans",
+  "item.croptopia.cheesy_asparagus": "Cheesy Asparagus",
+  "item.croptopia.chocolate_ice_cream": "Chocolate Ice Cream",
+  "item.croptopia.cooked_bacon": "Cooked Bacon",
+  "item.croptopia.eggplant_parmesan": "Eggplant Parmesan",
+  "item.croptopia.fruit_cake": "Fruit Cake",
+  "item.croptopia.grilled_eggplant": "Grilled Eggplant",
+  "item.croptopia.kiwi_sorbet": "Kiwi Sorbet",
```

Comment:
I would interpret this as a "sorbet from new zealand". I assume it's meant to be Kiwifruit Sorbet